docs/src/manual/exporting_to_jax.md

```diff
@@ -0,0 +1,144 @@
+# Exporting Lux Models to Jax (via StableHLO)
```

Comment:
Maybe say via Enzyme-JaX and Reactant?

Reply:
done

Reply:
It’s definitely worth having docs on this in Lux, but since a lot of this equally applies to general Julia code thanks to reactant, I wonder if it would make sense to add docs there and link to it from here (and of course show specific lux cases)

Reply:
And yeah lol we should do a package bump of enzyme jax, but haven’t had the chance to since my 3am commit adding that earlier lol.

Reply:
> It’s definitely worth having docs on this in Lux, but since a lot of this equally applies to general Julia code thanks to reactant, I wonder if it would make sense to add docs there and link to it from here (and of course show specific lux cases)

I will open a PR on reactant with an initial doc sketch.